packages/block-library/src/cover/block.json

```diff
@@ -32,6 +32,9 @@
 		"minHeight": {
 			"type": "number"
 		},
+		"minHeightUnit": {
+			"type": "string"
```

Comment:
I assume from this that it is preferred to let components like `CoverHeightInput` decide based on an undefined value, correct? Otherwise we might add a `default` value here.